app/javascript/article-form/actions.js

```diff
@@ -44,7 +44,7 @@ function processPayload(payload) {
   return neededPayload;
 }
 
-export function submitArticle(payload, clearStorage, errorCb, failureCb) {
+export function submitArticle(payload, clearStorage, successCb, errorCb) {
```

Comment:
Could these params become keyword arguments? With default empty callbacks, so that it can read something like:

```js
submitArticle({
 payload: blah blah
 onSuccess: () => this.setState({ published: true, submitting: false }),
})
```

so on and so forth.

Reply:
Good call @rhymes. I missed that.

Reply:
Yeah that would be nicer - I'll do that 👍